seed/challenges/jquery.json

```diff
@@ -676,7 +676,7 @@
       "title": "Target the Children of an Element Using jQuery",
       "difficulty": 3.14,
       "description": [
-        "Many HTML elements have <code>children</code> elements from which they <code>inherit</code> their properties.",
+        "Many HTML elements have <code>children</code> which <code>inherit</code> their properties from the HTML elements.",
```

Comment:
> from the HTML elements

Sounds better if it is written `from their HTML parent elements.` to further enforce that their is a relationship pattern.